csm/conf/etc/csm/csm.conf

```diff
@@ -128,6 +128,7 @@ CSM:
     username: "csm"
     password: ""
     password_decryption_key: "cortx"
+    session_backend: "database"
```

Comment:
Please change this to -> CSM>SESSION>storage

The values of storage can be 'local' or 'persistent'. 

You need to map consul to persistent and in-memory to local. 

In future if any new storage comes just map to either of the two

Reply:
updated